packages/api/endpoints/collections.js

```diff
@@ -17,10 +16,11 @@ const { indexCollection, deleteRecord } = require('../es/indexer');
  * @return {undefined}
  */
 function list(event, cb) {
-  const collection = new Collection(event);
-  collection.query().then(res => cb(null, res)).catch((e) => {
-    cb(e);
-  });
+  const collections = new models.Collection();
+  return collections
+    .scan()
```

Comment:
We should continue to use the ElasticSearch for returning results for the list endpoint, otherwise, none of our query parameters will work.